dspace-server-webapp/src/main/java/org/dspace/app/rest/BundleUploadBitstreamController.java

```diff
@@ -111,6 +124,30 @@ public ResponseEntity<RepresentationModel<?>> uploadBitstream(
             throw new UnprocessableEntityException("The InputStream from the file couldn't be read", e);
         }
 
+        // We do this before calling `updateBitstream` because that function calls `context.commit`
+        // Add suitable provenance
+        EPerson e = context.getCurrentUser();
+        String timestamp = DCDate.getCurrent().toString();
+        Item item = bundle.getItems().get(0);
```

Comment:
What if bundle.getItems is null?

Reply:
And maybe check if the fetched item is not null

Reply:
can there be a situation where the bundle does not have an assigned item? Shouldn't this situation be impossible? And shouldn't have one bundle only one item added?